docs/changes/1376.maintenance.md

```diff
@@ -0,0 +1 @@
+Maintenance PR to remove not needed flags and to only use metric files.
```

Comment:
Again - note that this is the only sentence the user will see. Is this enough information to understand the purpose of the changes?